cmd/root.go

```diff
@@ -51,6 +54,8 @@ func init() {
 	rootCmd.PersistentFlags().BoolVar(new(bool), "no-sort", false, "do no sort items")
 	rootCmd.PersistentFlags().BoolVar(&settings.SortByRequired, "sort-by-required", false, "sort items by name and print required ones first")
 
+	rootCmd.PersistentFlags().StringVar(&outputValuesPath, "output-values", "", "inject output values from `terraform output --json` into outputs")
```

Comment:
Ok now I see your point about this new flag, I then suggest to split that into two flags:

- `--output-values bool` : to indicate if output values need to get extract or not
- `--output-values-from string` : path of the json file to read the output from (defaults to `terraform-output.json`

Then we would be able to internally use terraform binary itself to extract json of output and parse it rather than forcing one extra step to the user. Although sometime a user might choose to do that extra step manually for whatever reason which will be fine too.

Reply:
I gave this some thought. What do you think of this alternative?

User passes `--output-values PATH` to specify a file from which to take output values.
OR
User passes `--output-values` to use the default PATH of `terraform-output.json` which is output by `terraform output --json > terraform-output.json`

The way I currently have set up has the default PATH set to an empty string, but I could easily change that to `terraform-output.json` and say that if the variable `outputValuesPath` is `""`, execute `terraform output --json > terraform-output.json`.

Two flags to do essentially the same thing sounds a bit :stuck_out_tongue_closed_eyes: to me, but it's entirely subjective.

Reply:
To be fair the two flags I mentioned don't do the same thing, `--output-values bool` is the indicator that we want to have outputs processed, and `--output-values-from string` is to specify from which JSON file we should read them, otherwise internally using `terraform` binary to attempt to fetch it within the execution of terraform-docs command.

Reply:
Refactored with your suggestions~